lib/hpke/test/userinfo_vc.cpp

```diff
@@ -214,14 +215,16 @@ TEST_CASE("UserInfoClaims Field Parsing")
 TEST_CASE("UserInfoClaims Edge Cases")
 {
   CHECK_THROWS_WITH(
-    UserInfoClaims::from_json({ { "updated_at", "42" } }),
+    UserInfoClaims::from_json(
+      nlohmann::json({ { "updated_at", "42" } }).dump()),
```

Comment:
Instead of constructing and dumping, you could just have the JSON strings here, using C++ raw strings to avoid quote ugliness.

```suggestion
    UserInfoClaims::from_json(R"({"updated_at", "42"})"),
```